examples/openapi-idl-client/openapi_idl_client.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/openapi;
+import ballerina/io;
+// Add the client import with OpenAPI specification.
+client "openapi.yaml" as foo;
+
+configurable string weatherAPIKey = ?;
+
+// You can use `@openapi:ClientConfig` annotation for customized your client.
+@openapi:ClientConfig {
+    tags: ["Current Weather Data"]
+}
+client "https://raw.githubusercontent.com/ballerina-platform/openapi-connectors/main/openapi/openweathermap/openapi.yaml" as weather;
```

Comment:
Shall we add the link of a specific tag/commit? The file in master may change.

Reply:
+1 , will update link with specific tag/commit 😃